doc/interpolation.rst

```diff
@@ -128,7 +128,7 @@ It is now possible to safely compute the difference ``other - interpolated``.
 Interpolation methods
 ---------------------
 
-We use :py:func:`scipy.interpolate.interp1d` for 1-dimensional interpolation and
+We use :py:class:`scipy.interpolate.interp1d` for 1-dimensional interpolation and
```

Comment:
```suggestion
We use :py:meth:`scipy.interpolate.interp1d` for 1-dimensional interpolation and
```
?

Reply:
`interp1d` is a class pretending to be a function:
```python
>>> type(scipy.interpolate.interp1d)
<class 'scipy.interpolate.interpolate.interp1d'>
```
so it is in the `py:class` section in `objects.inv`. To investigate, using
```bash
python -m sphinx.ext.intersphinx https://docs.scipy.org/doc/scipy/reference/objects.inv
```
helps a lot.